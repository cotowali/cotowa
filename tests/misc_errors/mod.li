module ns {
  var n = 0
  fn f() {}
}

var _ = n
f()
undefined_mod::f()
var _ = undefined_mod::n
