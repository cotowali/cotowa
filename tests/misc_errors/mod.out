error: mod.li 6,9: variable `n` is not defined
    6 | var _ = n
      |         ^
error: mod.li 7,1: function `f` is not defined
    7 | f()
      | ^^^
error: mod.li 8,1: module `undefined_mod` is not defined
    8 | undefined_mod::f()
      | ^^^^^^^^^^^^^
error: mod.li 9,9: module `undefined_mod` is not defined
    9 | var _ = undefined_mod::n
      |         ^^^^^^^^^^^^^
